app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt

```diff
@@ -317,9 +318,30 @@ class PlayerActivity : AppCompatActivity() {
                 return true
             }
         })
+        // Handles scale/zoom gesture
+        val zoomGestureDetector = ScaleGestureDetector(this, object : ScaleGestureDetector.OnScaleGestureListener {
+            override fun onScaleBegin(detector: ScaleGestureDetector): Boolean = true
+
+            override fun onScale(detector: ScaleGestureDetector): Boolean {
+                val scaleFactor = detector.scaleFactor
+                if (abs(scaleFactor - 1f) > 0.01f) {
+                    playerView.resizeMode = if (scaleFactor > 1) AspectRatioFrameLayout.RESIZE_MODE_ZOOM else AspectRatioFrameLayout.RESIZE_MODE_FIT
+                }
```

Comment:
```suggestion
                if (abs(detector.scaleFactor - 1f) > 0.01f) {
                    playerView.resizeMode = if (detector.scaleFactor > 1) AspectRatioFrameLayout.RESIZE_MODE_ZOOM else AspectRatioFrameLayout.RESIZE_MODE_FIT
                }
```

Reply:
Does that make such a difference? :thinking:
Actually, the extra variable ensures we get the same value for both accesses, and need less function calls.

Reply:
It shouldn't matter that much, it's more like a personal preference.